src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c

```diff
@@ -328,65 +337,65 @@ void func_80A0FD8C(EnFhgFire* this, GlobalContext* globalCtx) {
         globalCtx->envCtx.unk_E2[3] = 0x00;
     }
 
-    if (this->unk_150.x < 0x15) {
-        Math_ApproachZeroF(&this->unk_160, 1.0f, 45.0f);
+    if (this->timer <= 20) {
+        Math_ApproachZeroF(&this->alpha, 1.0f, 45.0f);
         Math_ApproachZeroF(&this->scale, 1.0f, 0.5f);
     } else {
-        Math_ApproachF(&this->scale, this->unk_18C, 0.5f, 3.0f);
+        Math_ApproachF(&this->scale, this->burstScale, 0.5f, 3.0f);
     }
 
     Actor_SetScale(&this->actor, this->scale);
-    if (3.0f < this->unk_18C) {
+    if (this->burstScale > 3.0f) {
         Collider_UpdateCylinder(&this->actor, &this->collider);
         if (player->invincibilityTimer == 0) {
             CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
         }
     }
 
-    if (this->unk_150.x == 0) {
+    if (this->timer == 0) {
         Actor_Kill(&this->actor);
         globalCtx->envCtx.unk_E1 = 0;
     }
-    if (this->unk_1FE != 0) {
-        this->unk_1FE--;
-        this->unk_1FC = 1;
-        Math_ApproachF(&this->unk_200, 40.0f, 0.3f, 10.0f);
+    if (this->unkTimer != 0) {
+        this->unkTimer--;
+        this->unkFlag = true;
+        Math_ApproachF(&this->unkFloat, 40.0f, 0.3f, 10.0f);
     } else {
-        Math_ApproachZeroF(&this->unk_200, 1.0f, 5.0f);
-        if (this->unk_200 == 0.0f) {
-            this->unk_1FC = 0;
+        Math_ApproachZeroF(&this->unkFloat, 1.0f, 5.0f);
+        if (this->unkFloat == 0.0f) {
+            this->unkFlag = false;
         }
     }
 
     // Related to scene draw config 30, only used in BossGanon_Update and
     // loaded in z_kankyo
-    D_8015FCF0 = this->unk_1FC;
+    D_8015FCF0 = this->unkFlag;
     D_8015FCF8 = this->actor.world.pos;
-    D_8015FD06 = this->unk_200;
+    D_8015FD06 = this->unkFloat;
     D_8015FD08 = 10.0f;
     D_8015FD0C = 0;
```

Comment:
this is related to lensflare. can send you more info if you want to document now

Reply:
sure